.php_cs.dist

```diff
@@ -1,18 +1,31 @@
 <?php
 
-$header = trim('This code is licensed under the BSD 3-Clause License.'.substr(file_get_contents('LICENSE'), strlen('BSD 3-Clause License')));
+declare(strict_types=1);
 
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
 
-$finder = \PhpCsFixer\Finder::create()
+$header = trim(
+    sprintf(
+        'This code is licensed under the BSD 3-Clause License.%s',
+        substr(
+            file_get_contents('LICENSE'),
+            strlen('BSD 3-Clause License')
+        )
+    )
+);
+
+$finder = Finder::create()
```

Comment:
What do you think about 

```diff
diff --git a/.php_cs.dist b/.php_cs.dist
index 436c2027..21e60be3 100644
--- a/.php_cs.dist
+++ b/.php_cs.dist
@@ -16,11 +16,19 @@ $header = trim(
 );

 $finder = Finder::create()
-    ->in('src')
-    ->in('tests')
-    ->in('bin')
-    ->exclude('Fixtures')
+    ->ignoreDotFiles(false)
+    ->in(__DIR__)
+    ->exclude([
+        '.ci',
+        '.composer',
+        '.github',
+        'build',
+        'devTools',
+        'resources',
+        'tests/Fixtures',
+    ])
     ->name('*php')
+    ->name('.php_cs.dist')
     ->name('infection')
     ->name('infection-debug')
```

?

Then this file here will be automatically fixed as well!

Reply:
Done :)